src/Utilities/generateConfluenceMarkdown.ts

```diff
@@ -25,10 +25,20 @@ export function generateConfluenceMarkdown(
             const ageInWholeDays = Math.round(ageInDays);
 
             if (ageInWholeDays < maxIssuesAge) {
-                markdownArray.push(`|[${issue.title}|${issue.url}]|${ageInWholeDays}&nbsp;days|`);
+                const cleanedTitleMarkdown = cleanMarkdown(issue.title);
+                markdownArray.push(
+                    `|[${cleanedTitleMarkdown}|${issue.url}]|${ageInWholeDays}&nbsp;days|`
+                );
             }
         });
     }
 
     return markdownArray.join('\n');
 }
+
+function cleanMarkdown(issueTitle: string): string {
+    const removedParenthesis = issueTitle.replace(/{|}/g, '');
+    const cleanedMarkdown = removedParenthesis.replace(/\[?\]/, '').replace(/\[?\[/, ''); //Brute force to clean extra brackets
```

Comment:
It looks like this is removing square brackets. And ... doing something else I don't understand. I'm curious why this extra code is needed, when Damon seemed to have one line that did what he felt was necessary. 

We should add tests to verify each specific thing that we believe clean needs to do.